src/scaffoldmaker/utils/geometry.py

```diff
@@ -64,19 +64,20 @@ def getEllipseArcLength(a, b, angle1Radians, angle2Radians):
     else:
         return -length
 
-def updateEllipseAngleByArcLength(a, b, inAngleRadians, arcLength):
+def updateEllipseAngleByArcLength(a, b, inAngleRadians, arcLength, tol=1.0E-4):
```

Comment:
See above comment - we need to make this more precise.
As mentioned we could pass in exact perimeter and current distance around to restore precision.